app/views/software_choices_filter.scala.html

```diff
@@ -105,15 +116,14 @@ <h1 class="heading-xlarge">@messages("softwareChoices.filter.title")</h1>
 
                 @search_filter_template(
                     FilterTemplateModel(
-                        messages("softwareChoices.filter.typeOfSoftware"),
-                        Seq(
-                            FilterFieldModel(form(accounting), messages("softwareChoices.filter.accountingSoftware")),
-                            FilterFieldModel(form(spreadsheets), messages("softwareChoices.filter.spreadSheets"))
+                        messages("softwareChoices.filter.price"),
+                        Seq(FilterFieldModel(form(free), messages("softwareChoices.filter.freeVersion"))
                         ),
-                        appConfig.features.typeOfSoftwareFilterEnabled()
+                    appConfig.features.typeOfSoftwareFilterEnabled()
```

Comment:
Should this be its own option?

Reply:
Agreed, unless there's a business reason why the Price filter should only be shown if the Type of Software filter is displayed, it should have its own toggle in the config